README.md

```diff
@@ -28,13 +28,6 @@ class Node {
 
 Any representation can be adapted to the needs of the algorithm:
 
-- Some algorithms need to associate data with each node in the graph and it will
-  be keyed by some type `K` that must work as a key in a `HashMap`. If nodes
-  implement `hashCode` and `==`, or if they are known to have one instance per
-  logical node such that instance equality is sufficient, then the node can be
-  passed through directly.
-  - `(node) => node`
-  - `(node) => node.id`
 - Algorithms which need to traverse the graph take a `children` function which
```

Comment:
`children` is a bit weird, right? How about `connections` or `edges`. `children` implies hierarchy, right?

These are positional params in all cases – we should be consistent, though.

Reply:
Edges sounds good to me